pubspec.yaml

```diff
@@ -9,17 +9,23 @@ dependencies:
   cached_network_image: ^3.2.3
   cloud_firestore: ^4.13.2
   cupertino_icons: ^1.0.2
+  dio: ^5.3.3
   firebase_auth: ^4.14.1
   firebase_core: ^2.24.2
   flutter:
     sdk: flutter
   flutter_hooks: ^0.20.4
+  flutter_rating_bar: ^4.0.1
```

Comment:
Please do not use the old package. The last update was 21 months ago.

Reply:
@mafreud Not updated because it doesn't have new features, I think there's no problem

Reply:
I don't think so. If you go to the issue page, you can find 37 issues and 12 PRs unaddressed.

Reply:
@mafreud i removed this package, and add new custom widget